workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/enums/Order.ts

```diff
@@ -15,8 +15,10 @@ export enum ORDER_CUSTOM_FIELDS {
 
 export enum ORDER_TYPES {
 	ADDONS = 'ADDONS',
-	DXPAPP = 'DXPAPP',
+	CLIENTEXTENSION = 'CLIENTEXTENSION',
```

Comment:
Hey @CalebJamesH I understand that you followed the pattern here, but this looks really strange, maybe a good opportunity to change.

Apply the same for all other places, thanks

```suggestion
	CLIENTEXTENSION = 'CLIENTEXTENSION',
```

```suggestion
	CLIENT_EXTENSION = 'CLIENT_EXTENSION',
```